apps/condo/lang/en.json

```diff
@@ -255,8 +255,14 @@
   "pages.condo.error.Description": "We are carrying out technical work. Please come back a little later.",
   "pages.condo.contact.PageTitle": "Contacts",
   "pages.condo.contact.ImportTitle": "contacts",
-  "pages.condo.property.EmptyBuildingDescription": "Fill required inputs and we will visualize it",
-  "pages.condo.property.EmptyBuildingHeader": "Describe the house",
+  "pages.condo.property.EmptyBuildingBlock.view.EmptyBuildingDescription": "Map creation may be done by two ways:\nmanually by filling some property parameters or ",
+  "pages.condo.property.EmptyBuildingBlock.edit.EmptyBuildingDescription": "This is at upper right corner of the screen\nThen everything is intuitive \uD83D\uDE0A",
```

Comment:
`It's in the upper right corner of the screen\nNext - everything is intuitive-understandable`